.github/workflows/ci.yml

```
@@ -280,7 +280,7 @@ jobs:
     needs: prepare-base
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9, "3.10"]
```

Should we pin versions exactly like zigpy?
```suggestion
        python-version: ['3.8.14', '3.9.14', '3.10.7']
```
https://github.com/zigpy/zigpy/pull/1050/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR21

---

Yes

---

Done.